cro-clib/src/lib.rs

```diff
@@ -0,0 +1,245 @@
+pub mod bindings;
+use crate::bindings::{ApiContext, ApiResult, Buf, HDWallet, MAX_STRING_LENGTH};
+use chain_core::init::network::init_chain_id;
+use client_common::error::ErrorKind;
+use client_common::SecKey;
+use client_core::types::WalletKind;
+use client_core::wallet::{DefaultWalletClient, WalletClient};
+use client_core::HDSeed;
+use client_core::Mnemonic;
+use secstr::SecUtf8;
+use std::ffi::CStr;
+use std::str::FromStr;
+
+use client_common::storage::MemoryStorage;
+#[derive(Default)]
+pub struct ApiInfo {
+    pub chain_id: String,
+}
+static mut G_DB: Option<MemoryStorage> = None;
+
+/// # Safety
+/// this function should not be called inside rust
+pub unsafe fn get_string(src: *const ::std::os::raw::c_char) -> String {
```

Comment:
Do we need to make this `pub`?

Reply:
for now, no, but later , if we use separate files, it's necessary.